packages/wix-ui-tpa/src/components/Checkbox/Checkbox.visual.tsx

```diff
@@ -58,9 +58,16 @@ visualize('Checkbox', () => {
 
   story('suffix', () => {
     snap(
-      `${CheckboxTheme.Box} / suffix`,
+      `${CheckboxTheme.Box} / suffix (string)`,
       <CheckboxVisual theme={CheckboxTheme.Box} suffix="$50,000" />,
     );
+    snap(
+      `${CheckboxTheme.Box} / suffix (reactNode)`,
+      <CheckboxVisual
+        theme={CheckboxTheme.Box}
+        suffix={<span>$50,000</span>}
```

Comment:
Just to make it to something we can notice the difference :) 
```suggestion
        suffix={<span style={{backgroundColor: 'red', padding: '3px'}}>$10</span>}
```